general/solomiia_tymoshchuk/lecture2/_1/Calculator.java

```diff
@@ -0,0 +1,73 @@
+package general.solomiia_tymoshchuk.lecture2._1;
```

Comment:
please move all files to the folder java

the package name should be just `lecture2._1`, it shouldn't include to project root folder